### PR TITLE
Added Chat Communication

### PR DESCRIPTION
**Pull request by darbrius**

### CODE REVIEW

**Review by rzayevan — APPROVED** [review]

File: poker-pals/src/components/pokerComponents/Chat.vue [hunk 1]
```diff
@@ -113,7 +105,19 @@
             },
             sendMessage: function (event) {
                 event.preventDefault(); // prevent page reload
-                // TODO: Send message to server
+                if (this.message !== "") {
```
Comment (chankk) [inline poker-pals/src/components/pokerComponents/Chat.vue]: You might want to check for other forms of white space.

File: poker-pals/src/components/pokerComponents/Chat.vue [hunk 2]
```diff
@@ -82,28 +82,20 @@
...
-                    { id: 11, senderID: 3000, name: 'Mark123', message: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.' },
-                    { id: 12, senderID: 3000, name: 'Luke854', message: 'f' },
-                    { id: 13, senderID: 1000, name: 'Luke854', message: 'f' },
-                    { id: 14, senderID: 1000, name: 'Luke854', message: 'f' },
-                ],
+                items: [],
             }
         },
+        mounted: function() {
+            var self = this;
```
Comment (chankk) [inline poker-pals/src/components/pokerComponents/Chat.vue]: let?

File: poker-pals/src/components/pokerComponents/Chat.vue [hunk 3]
```diff
@@ -82,28 +82,20 @@
...
+            this.socket.on('messageSentSuccessful', function(msg){
+                let you = msg.senderID === self.$parent.userID;
+                self.addMessage({id: msg.id, senderID: msg.senderID, name: msg.name, you: you, message: msg.message});
```
Comment (chankk) [inline poker-pals/src/components/pokerComponents/Chat.vue]: maybe multi-line the object for readability?

**Review by aidan-lee — APPROVED** [review]

**Review by chankk — CHANGES_REQUESTED** [review]
Concise & nice overall, I mainly want you to double-check that empty/white space messages aren't sent.

**Review by chankk — APPROVED** [review]

**Pull request merged**